LessonOneHomework/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace LessonOneHomework
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool continueLoop = true;
+            while(continueLoop)
+            {
+                (string firstName, string surname) = GetUserName();
+                Console.Write("What is your age? ");
+                int age = int.Parse(Console.ReadLine());
+                (double height, double weight) = GetHeightandWeight();
+                double bmi = CalculateBMI(height, weight);
+                Console.WriteLine($"{firstName} {surname} is {age} years old, his weight is {weight} and his height is {height}.");
+                Console.WriteLine($"Their BMI is {bmi}.");
+                Console.Write("Another person? (y/n)");
+                string looper = Console.ReadLine();
+                continueLoop = looper.ToLower() == "n" ? false : true;
+            }
+        }
+
+        private static (string, string) GetUserName()
```

Comment:
We don't need to have a distinction between a firstname and lastname- no need to return 2 things.

Reply:
Admittedly, it's a job for a class and not tuples, but we haven't covered classes yet so I just did tuples for fun. 

Also, I only made a distinction between firstname and lastname because the requirements listed them distinctly.